app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1 @@
+$warning_orange: #F89E54;
```

Comment:
In general components should own their own variables, and should be specified with hyphens (rather than underscores). [There's an example of this pattern in use here.](https://github.com/moneyadviceservice/rad/tree/master/app/assets/stylesheets/components/notice) And a little explanation [here](https://github.com/moneyadviceservice/rad/pull/198#discussion-diff-31034253) and [here](https://github.com/moneyadviceservice/rad/pull/198#discussion-diff-31034281).